### ISSUE TITLE
Setting threshold

### ISSUE THREAD

**Issue by BernardinD** [junqiangchen/Unet2d#1]
Hi, I have implemented you code and get results but I don't have any control over the threshold. Could you tell me how I could gain this control?

**Comment by junqiangchen** [comment]
> Hi, I have implemented you code and get results but I don't have any control over the threshold. Could you tell me how I could gain this control?

there are four trained superparameters that can controlled by you，learning rate、droupout value，training epoches，batch size.In the file glandceilunet2dtest.py.
